bin/lamassu-register

```diff
@@ -1,26 +1,44 @@
 #!/usr/bin/env node
 
-const login = require('../lib/admin/login')
+const authentication = require('../lib/new-admin/graphql/modules/authentication')
 const options = require('../lib/options')
 
 const name = process.argv[2]
+const role = process.argv[3]
 const domain = options.hostname
 
 if (!domain) {
   console.error('No hostname configured in lamassu.json')
   process.exit(1)
 }
 
-if (!name) {
-  console.log('Usage: lamassu-register <username>')
+if (!name || !role) {
+  console.log('Usage: lamassu-register <username> <role>')
   process.exit(2)
 }
 
-login.generateOTP(name).then(otp => {
+const emailRegex = /^(([^<>()[\]\\.,;:\s@\"]+(\.[^<>()[\]\\.,;:\s@\"]+)*)|(\".+\"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/
```

Comment:
are we just testing basic email addresses here or is it important to be 99% sure the email is valid?

we could go simpler: `/^[^\s@]+@[^\s@]+\.[^\s@]+$/`
but if we want to make sure everything is 99.99% ok we could use a tried and true expression: https://emailregex.com/

Reply:
The more valid it is, the better, I think. Formik is checking for email formats, not sure how robust their check is though